Modules/+Drivers/ArduinoServo.m

```diff
@@ -0,0 +1,64 @@
+classdef ArduinoServo < Modules.Driver
+    %ARDUINOSERVO Connects with a hwserver Arduino and uses one pin to control a servo.
+    %
+    % Call with the 1) IP of the host computer (singleton based on ip), and 2) the (integer) pin.
+    
+    properties (Constant)
+        hwname = 'Arduino';
+    end
+    properties (SetAccess=immutable)
+        connection
+        pin
+    end
+    properties (GetObservable, SetObservable)
+        angle = Prefs.Double(NaN, 'min', 0, 'max', 180, 'set', 'set_angle', 'allow_nan', true);
+    end
+    methods(Static)
+        function obj = instance(ip, pin)
+            mlock;
+            pin = round(pin);
```

Comment:
Why round? It should error if called with a non-integer - likely the user was confused when they called.